solutions/search/inference-api/alibabacloud-ai-search-inference-integration.md

```diff
@@ -43,7 +43,7 @@ Creates an {{infer}} endpoint to perform an {{infer}} task with the `alibabaclou
 ## {{api-request-body-title}} [infer-service-alibabacloud-ai-search-api-request-body] 
 
 `chunking_settings`
-:   (Optional, object) Chunking configuration object. Refer to [Configuring chunking](https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html#infer-chunking-config) to learn more about chunking.
+:   (Optional, object) Chunking configuration object. Refer to [Configuring chunking](https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-inference) to learn more about chunking.
```

Comment:
Unfortunately this link doesn't make sense anymore because that information is not in the new reference yet

This holds for all of these ones in the `inference-api` directory.

cc @szabosteve @charlotte-hoblik 

we might just have to remove these links entirely for the moment until that chunking information is restored somewhere centrally? LMKWYT

Reply:
https://elastic.slack.com/archives/C059JCWTPMX/p1739268759792579

Reply:
@colleenmcginnis feel free to ignore this and we'll create a TODO to handle it next week :)